portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Leon's Gallery</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
```

Comment:
To improve the website performance better, you can read an article below
https://flaviocopes.com/javascript-async-defer/

Reply:
Alright awesome will look into this, thanks!

Reply:
The <script> here can also use defer, though I see that this page does not have any content.
Maybe even remove the <script> tag since the onLoad is not meant to be applied here?

Reply:
Oh right missed that out thanks! I think I will just use defer so that I can add on to the gallery page in the future